--- lmdb-LMDB_0.9.21/libraries/liblmdb/Makefile~	2017-06-01 16:51:10.000000000 +0000
+++ lmdb-LMDB_0.9.21/libraries/liblmdb/Makefile	2017-09-17 19:55:54.799544941 +0000
@@ -27,10 +27,10 @@
 LDLIBS	=
 SOLIBS	=
 SOEXT	= .so
-prefix	= /usr/local
+prefix	= /usr
 exec_prefix = $(prefix)
 bindir = $(exec_prefix)/bin
-libdir = $(exec_prefix)/lib
+libdir = $(exec_prefix)/lib64
 includedir = $(prefix)/include
 datarootdir = $(prefix)/share
 mandir = $(datarootdir)/man
